### PR TITLE
Iterated cartesian product is closed by permutation

### PR DESCRIPTION
**Pull request by VictorBlanchi**
I have shown that the 3 definitions of iterated cartesian product of types is closed by permutation.

### CODE REVIEW

**Review by fredrik-bakke — APPROVED** [review]
I just have some comments on wording ^^

**Comment by fredrik-bakke** [comment]
Thanks for addressing my comments! I'll merge the PR once you fix the last typo. 🙂

File: src/foundation/iterated-cartesian-product-types.lagda.md [hunk 1]
```diff
@@ -71,9 +84,9 @@ iterated-product-lists {l} nil = raise-unit l
 iterated-product-lists (cons A p) = A × iterated-product-lists p
 ```
 
-## Property
+## Properties
 
-### Equivalence between the first two definitions
+### The definitions using recursion and `Π-type` are equivalent
```
Comment (EgbertRijke) [inline src/foundation/iterated-cartesian-product-types.lagda.md:89]: `### The definitions using recursion and `Π`-types are equivalent

Comment (fredrik-bakke) [reply src/foundation/iterated-cartesian-product-types.lagda.md]: Weird, I wrote a comment pointing this out, but it doesn't show up here.

https://github.com/UniMath/agda-unimath/commit/85d5d759286ca68a182652f8bda2450ffc2156ff#r112871920

**Pull request merged**